src/components/DownloadFileButton.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+import {omit} from 'lodash';
+import FileSaver from 'file-saver';
+import cx from 'classnames';
+
+export default class DownloadFileButton extends Component {
```

Comment:
I don't think this controls is needed at all - there is not so much logic. Actually it's just saveFile method which can be passed as onClick